src/YesSql.Abstractions/IQuery.cs

```diff
@@ -111,6 +111,12 @@ public interface IQuery<T> where T : class
         /// Adds a Group By clause to the query for fold documents when an index returns multiple records.
         /// </summary>
         IQuery<T> NoDuplicates();
+
+        /// <summary>
+        /// When set, the fetched objects will not be cached internally.
+        /// </summary>
+        /// <returns></returns>
```

Comment:
```suggestion
/// <remarks> When called, the fetched objects will not be cached internally.</remarks>
```

Reply:
sorry I don't like the new comment.

Reply:
No problem, at least remove un empty returns tag, also there's no set in the method

Reply:
@hishamco updated,